common/components/test/utils.test.js

```diff
@@ -196,4 +198,11 @@ describe("utils", () => {
     word = utils.pluralize("apple", "apples", 5);
     expect(word).toEqual("apples");
   }); 
+
+  test("guard.duplicateInput", () => {
+    _.debounce = jest.fn();
+    const testFunc = () => "test";
+    Guard.duplicateInput(testFunc);
+    expect(_.debounce).toBeCalledWith(testFunc, 1000, {leading: true});
```

Comment:
I would instead test that testFunc was called only once after multiple invocations of Guard.duplicateInput's return function.